toxcore/tox.h

```diff
@@ -473,7 +465,8 @@ typedef enum Tox_Log_Level {
     TOX_LOG_LEVEL_WARNING,
 
     /**
-     * Severe unexpected errors caused by external or events_alloc inconsistency.
+     * Severe unexpected errors caused by external or events_alloc
```

Comment:
events_alloc should be "internal". This was a mistake made by me and not caught in review.

Reply:
Should be fixed now, please check.